app/views/pages/profile.html.erb

```diff
@@ -0,0 +1,50 @@
+<div class="container mt-5">
+  <!-- Content here -->
+  <div class="d-flex flex-row bd-highlight mb-3 justify-content-between">
+    <div class="d-flex">
+      <img src="https://source.unsplash.com/X6Uj51n5CE8/150x150" class="img-thumbnail" alt="Profile Picture">
```

Comment:
This is still a static image, place replace this by the profile image with the right class and the link to the image